dfastmi/gui/dialog_view_model.py

```diff
@@ -359,8 +359,15 @@ def load_configuration(self, filename: str) -> bool:
                 self._reference_files[cond_discharge] = section.get("Reference", "")
                 self._measure_files[cond_discharge] = section.get("WithMeasure", "")
 
-        self.current_branch = self.model.rivers.get_branch(self.model.branch_name)
-        self.current_reach = self.current_branch.get_reach(self.model.reach_name)
+        branch = self.model.rivers.get_branch(self.model.branch_name)
```

Comment:
why do you want to set a river branch and reach which is not in the analysis configuration file, would None be sufficient and show nothing in the GUI?

Reply:
The default (internal) config definition doesn't have a branch and reach set. The branch and reach are dropdown fields, so they can't be None/empty. I'm perfectly fine with defaulting to the first entry in the respective lists and that's the purpose of this code.

Reply:
Discussed with Ralph.